src/_nebari/stages/kubernetes_ingress/template/modules/kubernetes/ingress/main.tf

```diff
@@ -171,6 +171,46 @@ resource "kubernetes_service" "traefik_internal" {
   }
 }
 
+
+
+
+resource "kubernetes_persistent_volume_claim" "persistent-volume-claim" {

```

Comment:
"persistent-volume-claim" - use _ and specify a bit more in the name, traefik_ingress_pvc should work